resources/seccomp/x86_64-unknown-linux-musl.json

```diff
@@ -609,6 +609,9 @@
                         "comment": "FIONBIO"
                     }
                 ]
+            },
+            {
+                "syscall": "recvmsg"
```

Comment:
Why is this syscall needed ?

Reply:
micro-http replaced `read` with `recv_with_fd` where they handle bytes streams. The seccomp filter will kill the microvm because the new syscall is not approved(yet)

Reply:
Ok. Can you also add a comment about this please ?

Reply:
It's due to [this commit in micro-http repo](https://github.com/firecracker-microvm/micro-http/commit/3c4cc3aa9170eeb7c43fe1019497580ec0172afd), where `read()` has been replaced by `recv_with_fd()`.